sdk/python/feast/registry.py

```diff
@@ -830,6 +830,10 @@ def to_dict(self, project: str) -> Dict[str, List[Any]]:
             registry_dict["savedDatasets"].append(
                 MessageToDict(saved_dataset.to_proto())
             )
+
+        registry_dict["infra"].append(
```

Comment:
this is breaking in `test_cli.py` due to the following:
```
registry_specs = {
    key: [fco["spec"] for fco in value]
    for key, value in registry_dict.items()
}
```
maybe we can change the test to 
```
registry_specs = {
    key: [fco["spec"] if "spec" in fco else fco for fco in value]
    for key, value in registry_dict.items()
}
```